app/libs/classes/application_answer/zeep_person_info.py

```diff
@@ -0,0 +1,133 @@
+        self.find_by_tag = find_by_tag
+        self.person_type = person_type
+
+    def create(self):
+        creater_mapping = {
+            'client': self._create_applicant,
+            'partner': self._create_applicant,
+            'children': self._create_children,
+        }
+
+        return creater_mapping[self.person_type]() or {}
+
+    def _create_applicant(self):
+        applicantAnswers: List[ApplicationAnswer] = self.application_answer_collection.filter_by_tags(
+            tags=[self.find_by_tag])
+
+        if not applicantAnswers:
+            return None
+
+        return {
+            self.person_type.upper(): {
+                'PNUMBER': self._get_value(answers=applicantAnswers, tags=['personalNumber']),
+                'FOREIGNCITIZEN': False,
+                'RESIDENCEPERMITTYPE': '',
+                'RESIDENCEPERMITDATE': '',
+                'CIVILSTATUS': 'G',
+                'ALTCIVILSTATUS': '',
+                'PHONENUMBERS': {
+                    'PHONENUMBER': {
+                        'TYPE': 'Mobiltelefon',
+                        'NUMBER': self._get_value(answers=applicantAnswers, tags=['telephone']),
+                        'SMS': False
+                    }
+                },
+                'EMAIL': {
+                    'EMAIL': self._get_value(answers=applicantAnswers, tags=['email']),
+                    'NOTIFY': False
+                },
+                'FNAME': self._get_value(answers=applicantAnswers, tags=['firstName']),
+                'LNAME': self._get_value(answers=applicantAnswers, tags=['lastName']),
+                'ADDRESSES': {
+                    'ADDRESS': {
+                        'TYPE': 'FB',
+                        'CO': '',
+                        'ADDRESS': self._get_value(answers=applicantAnswers, tags=['address']),
+                        'ZIP': self._get_value(answers=applicantAnswers, tags=['postalCode']),
+                        'CITY': self._get_value(answers=applicantAnswers, tags=['postalAddress'])
+                    }
+                }
+            }
+        }
+
+    def _create_children(self):
+        child_answers: List[ApplicationAnswer] = self.application_answer_collection.filter_by_tags(
+            tags=[self.find_by_tag, 'children'])
+        if not child_answers:
+            return None
+
+        child = [self._create_child(answers)
+                 for answers in self._create_grouped_answers(child_answers)]
+
+        return {
+            self.person_type.upper(): {
+                'CHILD': child if len(child) > 1 else child[0],
+            },
+        }
+
+    def _create_grouped_answers(self, answers: List[ApplicationAnswer]) -> List[List[ApplicationAnswer]]:
+        grouped_answers: List[List[ApplicationAnswer]] = []
+
+        for answer in answers:
+            group_tag: str = answer.get_tag_starting_with('group:')
+            group_index = int(group_tag.split(':')[1]) if group_tag else 0
```

Comment:
To comply with how some tags are made, `group_index` should probably check for the last part of the `group_tag` array instead (by using `[-1]`):

```python
group_index = int(group_tag.split(':')[-1]) if group_tag else 0
```